### PR TITLE
#645 add samples for charSequenceAssertions of api-fluent

### LINKED ISSUE #645: add samples for charSequenceAssertions of api-fluent
## Code related feature
See for the initial request: https://github.com/robstoll/atrium/issues/624

With this issue we want to add samples for charSequenceAssertions.kt of api-fluent

Following the things you need to do:
 
*api-fluent*
- [ ] create CharSequenceAssertionSamples in apis/fluent-en_GB/atrium-api-fluent-en_GB-common/src/test/kotlin/ch/tutteli/atrium/api/fluent/en_GB/samples
  - [ ] add one function per function in charSequenceAssertions.kt (see AnyAssertionSamples)
  - [ ] refer to the sample in the KDoc of the corresponding function in charSequenceAssertions via `@sample` (check out `toBe` in anyAssertions.kt to see how you have to refer to the sample)

## Your first contribution?
- Write a comment `I'll work on this` if you would like to take this issue over. 
  This way we get the chance to revise the description in case things have changed in the meantime, we might give you additional hints and we can assign the task to you, so that others do not start as well.
- See [Your first code contribution](https://github.com/robstoll/atrium/blob/master/.github/CONTRIBUTING.md#your-first-code-contribution) for guidelines.  
- Do not hesitate to ask questions here or to contact us via [Atrium's slack channel](https://kotlinlang.slack.com/team/U3DE1TXKP) if you need help
  ([Invite yourself](https://slack.kotlinlang.org/) in case you do not have an account yet).

### PR DESCRIPTION
**Pull request by jdornieden**
Resolve #645 
______________________________________
I confirm that I have read the [Contributor Agreements v1.0](https://github.com/robstoll/atrium/blob/master/.github/Contributor%20Agreements%20v1.0.txt), agree to be bound on them and confirm that my contribution is compliant.

### CODE REVIEW

**Comment by jdornieden** [comment]
@robstoll Build 11 failed with a weird exception on the first run. Flaky build?

**Comment by robstoll** [comment]
There is something strange going on with github actions currently. I don't know yet why

File: apis/fluent-en_GB/atrium-api-fluent-en_GB-common/src/test/kotlin/ch/tutteli/atrium/api/fluent/en_GB/samples/CharSequenceAssertionsSpec.kt [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+
+    @Test
+    fun contains() {
+        expect("ABC").contains("B")
+
+        expect("ABC123").contains("AB", 'C', 12)
+
+        // holds because `contains` does not search for unique matches
+        // use `contains.exactly(2).value("A")` to check if subject contains two "A"s
+        expect("ABC").contains("A", "A")
```
Comment (robstoll) [inline apis/fluent-en_GB/atrium-api-fluent-en_GB-common/src/test/kotlin/ch/tutteli/atrium/api/fluent/en_GB/samples/CharSequenceAssertionsSpec.kt:40]: nice :+1:

File: apis/fluent-en_GB/atrium-api-fluent-en_GB-common/src/test/kotlin/ch/tutteli/atrium/api/fluent/en_GB/samples/CharSequenceAssertionsSpec.kt [hunk 2]
```diff
@@ -0,0 +1,232 @@
+package ch.tutteli.atrium.api.fluent.en_GB.samples
+
+import ch.tutteli.atrium.api.fluent.en_GB.*
+import ch.tutteli.atrium.api.verbs.internal.expect
+import kotlin.test.Test
+
+class CharSequenceAssertionsSpec {
+
+    @Test
+    fun containsFeature() {
```
Comment (robstoll) [inline apis/fluent-en_GB/atrium-api-fluent-en_GB-common/src/test/kotlin/ch/tutteli/atrium/api/fluent/en_GB/samples/CharSequenceAssertionsSpec.kt:10]:
```suggestion
    fun containsBuilder() {
```

File: apis/fluent-en_GB/atrium-api-fluent-en_GB-common/src/test/kotlin/ch/tutteli/atrium/api/fluent/en_GB/samples/CharSequenceAssertionsSpec.kt [hunk 3]
```diff
@@ -0,0 +1,232 @@
...
+            .contains.atLeast(2).value("B")
+
+        fails {
+            expect("AAAAAA")
+                .contains.atMost(3).value("A")
+        }
+    }
+
+    @Test
+    fun containsNotFeature() {
```
Comment (robstoll) [inline apis/fluent-en_GB/atrium-api-fluent-en_GB-common/src/test/kotlin/ch/tutteli/atrium/api/fluent/en_GB/samples/CharSequenceAssertionsSpec.kt:24]:
```suggestion
    fun containsNotBuilder() {
```

**Review by robstoll — APPROVED** [review]
Two small things which are OK as they are but could be improved (e.g. in a next PR)

**Comment by robstoll** [comment]
@jdornieden many thanks also for this contribution, looks good :slightly_smiling_face: :+1: 
See review above, maybe something you can improve in a next PR

**Pull request merged**